### ISSUE TITLE
named.conf.erb assumes named.rfc1912.zones in hardcoded location

### ISSUE THREAD

**Issue by mainvoid** [thias/puppet-bind#61]
On at least Debian 8  this file resides in /etc/bind and not in /etc causing named.conf to point to a non-existing file, which in turn stops BIND from starting, giving a cryptic error of:

 /etc/bind/named.conf:61: missing ';' before '/'

Lots of loss of hair later, I noticed that the include path was incorrect. Manually changing it made BIND restart happily ...

**Comment by drequena** [comment]
Actually, I didn't even find the rfc1912 and named.ca files on Debian 8.
I had to remove the entries from named.conf.erb in order to have a functional DNS server.

**Comment by rendhalver** [comment]
I ran into a similar issue on Ubuntu.
I added named.ca and the rfc1912 zones files in my profile but ran into an issue where apparmor is complaining about the files location.
I am working on a fix that should resolve the issue.

**Comment by hybby** [comment]
I ran into this issue on Ubuntu 14.04 too.  I had to change the following in the`name.conf.erb` file to get BIND started...

```
-include "/etc/named.rfc1912.zones";
+include "/etc/bind/zones.rfc1918";
``

```
-    file "named.ca";
+    file "/etc/bind/db.root";
```

**Comment by khaefeli** [comment]
added a PR to solve this isseu @thias

**Comment by thias** [comment]
This should now be fixed in 1c9dbfc8fe1b16c3406f9ff01cf9dc02faf0057c. Also note that RFC1912 and RFC1918 are quite different, and after a quick look, it seems like the Debian file covering RFC1912 and the forward/reverse for loopback addresses and names is `/etc/bind/named.conf.default-zones` so I have included that one.

But I'm assuming most of you have moved on to better maintained modules anyway (sorry!).
